baremetal/metal3data_manager_test.go

```diff
@@ -3950,7 +3951,7 @@ type releaseAddressFromPoolFakeClient struct {
 
 func (f *releaseAddressFromPoolFakeClient) Delete(ctx context.Context, obj client.Object, opts ...client.DeleteOption) error {
 	if f.injectDeleteErr {
-		return fmt.Errorf("failed to delete for some weird reason")
+		return errors.New("failed to delete for some weird reason")
```

Comment:
Is some linter asking to use `github.com/pkg/errors` ? The project(`github.com/pkg/errors`) is archived and I think we should not use it, even remove it if being used anywhere in the project.

Reply:
Nope, fixed.